aws-alb-ingress-controller/templates/deployment.yaml

```diff
@@ -18,7 +18,13 @@ spec:
         {{- range $key, $value := .Values.podLabels }}
         {{ $key }}: {{ tpl $value $root | quote }}
         {{- end }}
-      annotations: {{- with .Values.podAnnotations }}{{- tpl (toYaml . ) $root | nindent 8 }}{{- end }}
+      annotations:
+        {{- if .Values.metrics.enabled }}
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "{{ .Values.metrics.port }}"
+        prometheus.io/path: "{{ .Values.metrics.path }}"
```

Comment:
@taishin 
It doesn't have the metrics port specified in the container port, will this work?

Reply:
In case the port number for metrics changes in the future, set it differently from containerPort. 
(Currently, support for alb-ingress-controller metrics is not stated in the documentation.)